resources/functionMap.php

```diff
@@ -271,7 +271,7 @@
 'array_diff_uassoc\'1' => ['array', 'arr1'=>'array', 'arr2'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest='=>'array|callable'],
 'array_diff_ukey' => ['array', 'arr1'=>'array', 'arr2'=>'array', 'key_comp_func'=>'callable(mixed,mixed):int'],
 'array_diff_ukey\'1' => ['array', 'arr1'=>'array', 'arr2'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest='=>'array|callable(mixed,mixed):int'],
-'array_fill' => ['array', 'start_key'=>'int', 'num'=>'int', 'val'=>'mixed'],
+'array_fill' => ['array', 'start_index'=>'int', 'count'=>'int', 'value'=>'mixed'],
```

Comment:
adjusted the param names to the ones named on php.net: https://www.php.net/manual/en/function.array-fill.php